source/features/hide-tips.css

```diff
@@ -23,8 +23,8 @@ a.tabnav-extra[href$='mastering-markdown/'] {
 	display: none !important;
 }
 
-/* Remove random protip at the bottom of the page */
-.protip {
+/* Remove random protip at the bottom of conversations lists */
+.paginate-container + .text-center:last-child {
```

Comment:
I haven't seen those tips anywhere else, apart from one at the very bottom of the dashboard (not really worth fixing, plus its DOM is annoying).

Also, worth keeping the old selector for GHE?

Reply:
> Also, worth keeping the old selector for GHE?

No